src/interaction/InteractionTrackingData.js

```diff
@@ -36,7 +36,7 @@ export default class InteractionTrackingData
 
     /**
      * @readonly
-     * @type {number} Unique pointer id of the event
+     * @return {number} Unique pointer id of the event
```

Comment:
Do not agree with this one. Getters are functions to replicate member variables. It should say member rather than type. Definitely not return